src/Exception/KeepaliveException.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\MessengerHeartbeatBundle\Exception;
+
+use Symfony\Component\Messenger\Exception\StopWorkerException;
+
+class KeepaliveException extends StopWorkerException
```

Comment:
zamieniłem jednak HeartbeatConnectionLostException na KeepaliveException.
Dodatkowo rozszerza on StopWorkerException, więc jak coś będzie nie tak na keepalive to zamykam workera. Wychodzę z założenia, że jak nie działa to nie ma co procesować na tym konkretnym.